charts/external-dns/values.yaml

```
@@ -279,6 +279,8 @@ provider:
       scrapeTimeout:
       metricRelabelings: []
       relabelings: []
+    service:
+      metricsPort: 8080
```

```suggestion
    service:
      # -- Webhook metrics port for the service.
      metricsPort: 8080
```
Values should be annotated for `helm-docs`, feel free to make this better. You'll need to run `helm-docs` locally to update the README before the automation will pass.

Also this could/should also be moved up above `serviceMonitor`.

---

You got it 👍
I have pushed a new commit with the changes